src/test/ui/consts/const-size_of-cycle.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-musl
+// ignore-x86
```

Comment:
I don't see this discussed explicitly. What is causing the problem these `ignore`s are, well, ignoring? Was an issue ever opened to track it.

This showed up again in https://github.com/rust-lang/rust/pull/58140#issuecomment-473194939, and while I'm fine with copying the two directives from here, this seems like a bug that should be tracked, if not investigated right away.

Reply:
The failure [happened here](https://travis-ci.org/rust-lang/rust/jobs/424637306) and I wasn't really interested in debugging this, but I agree I should have opened up an issue to track it.